test/mpi/threads/comm/dup_leak_test.c

```diff
@@ -15,21 +15,19 @@
 #include "mpitest.h"
 #include "mpithreadtest.h"
 
-#ifndef NITER
-#define NITER 12345
-#endif /* ! NITER */
-
 #ifndef NTHREADS
 #define NTHREADS 4
 #endif /* ! NTHREADS */
 
+int num_iter = 12345;
```

Comment:
I think the MTestArg macros will raise an error if the argument is not supplied on the command line, so this static initialization might be confusing to readers.

Reply:
Fixed.

Reply:
I think it is useful to have an `MTestArg_with_default`.

Reply:
Yes, after working on the GPU buffer tests, I agree. It would be nice to default to host memory allocation.